crates/gossip/src/gossip.rs

```diff
@@ -502,11 +513,19 @@ mod test {
     }
 
     impl TestGossipFactory {
-        pub async fn create(space: SpaceId) -> TestGossipFactory {
+        pub async fn create(
+            space: SpaceId,
+            bootstrap: bool,
+        ) -> TestGossipFactory {
             let mut builder =
                 default_test_builder().with_default_config().unwrap();
             // Replace the core builder with a real gossip factory
             builder.gossip = K2GossipFactory::create();
+
+            if !bootstrap {
```

Comment:
Gives us a way to disable bootstrap so that gossip doesn't start until we manually populate one of the peer stores